www/templates/www/user.html

```diff
@@ -5,12 +5,14 @@
 {% block content %}
 <div class="container">
   <h1>
-    {% trans 'User data for' %} {{ userdetails.first_name }} {{ userdetails.last_name }}
+    {% trans 'User data for' %} {{ userdetails }}
   </h1>
 
   {% if user.is_staff %}
     <p>
       <a href="{% url 'updateuser' userdetails.id %}">Update user's data</a> |
+      <a href="{% url 'users/delete' userdetails.id 0 %}">D</a>elete user - keep account |
```

Comment:
I'm guessing the link is for only the D letter for fat finger protection?

Reply:
Yep